.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+on:
```

Comment:
this is a release for npm. here we just need to release on GitHub, take a look at [this](https://github.com/gagoar/use-herald-action/blob/master/.github/workflows/release.yml)